docs/StardustDocs/topics/convert.md

```diff
@@ -8,7 +8,7 @@ convert { columnsSelector }
     .with { rowExpression } | .perRowCol { rowColExpression } | .withValue(value)  | to<Type>() | to { colExpression }
 
 rowExpression = DataRow.(OldValue) -> NewValue
-rowColExpression = DataRow.(DataColumn) -> NewValue
+rowColExpression = (DataRow, DataColumn) -> NewValue
```

Comment:
Maybe we should fix the type instead?

Reply:
no I think the type is right. Usually we have the context as `this` and then the specific thing you mention as `it`. Here there are two specific things. They cannot both be `it`, so they should be manually named `row` and `col` by the user of the lambda :)

Reply:
Right, two parameters here make more sense